code/_onclick/hud/parallax.dm

```diff
@@ -45,12 +45,14 @@ var/list/parallax_icon[(GRID_WIDTH**2)*3]
 	0,0,0,0,
 	0,0,0,0,
 	0,0,0,0,
-	1,1,1,1,
-	0,0,0,0)
+	0,0,0,0,
```

Comment:
Imagine that every pixel in the image has four values, r - red, g - green, b - blue, and a - alpha (transparency)

Now imagine you make a 1x5 matrix out of them, [red, green, blue, alpha, nothing].

A color matrix gets multiplied to that original matrix to make the final color of every pixel. The color matrix itself is 5x5 so that it multiplies with the 1x5 to make a 1x5 matrix, assuming you remember the rules of simple matrix multiplication.

What each part of the color matrix represents is how they continbute to the final value
Stealing from byond's ref
```
                |rr rg rb ra 0|
                |gr gg gb ga 0|
[r g b a 255] x |br bg bb ba 0| = [r' g' b' a' 255]
                |ar ag ab aa 0|
                |cr cg cb ca 1|
```

Meaning that each final value is calculated like this
```
new_red   = red * rr + green * gr + blue * br + alpha * ar + 255 * cr
new_green = red * rg + green * gg + blue * bg + alpha * ag + 255 * cg
new_blue  = red * rb + green * gb + blue * bb + alpha * ab + 255 * cb
new_alpha = red * ra + green * ga + blue * ba + alpha * aa + 255 * ca
```

Of course byond supports several truncated forms of this, including a 3x3, 4x3, 4x4, or 5x4 color matrix depending on what you want it to do. Also since the last value of the 5x5 and 1x5 matrix above is *totally fucking useless*, you can truncate that and give it a 5x4 matrix instead.